src/openforms/registrations/contrib/objects_api/config.py

```diff
@@ -191,7 +203,35 @@ def to_internal_value(self, data: Any) -> Any:
 
         return super().to_internal_value(new_data)
 
+    def _set_group_during_import(self, attrs: dict[str, Any]) -> None:
+        if (
+            self.context.get("is_import", False)
+            and attrs.get("objects_api_group") is None
+        ):
+            existing_group = (
+                ObjectsAPIGroupConfig.objects.order_by("pk")
+                .exclude(
+                    Q(objects_service=None)
+                    | Q(objecttypes_service=None)
+                    | Q(drc_service=None)
+                    | Q(catalogi_service=None)
+                )
+                .first()
+            )
+            if existing_group is None:
+                raise serializers.ValidationError(
+                    {
+                        "objects_api_group": _(
+                            "You must create a valid Objects API Group config (with the necessary services) before importing."
+                        )
+                    }
+                )
+            else:
+                attrs["objects_api_group"] = existing_group
```

Comment:
if there are multiple candidate groups, we should grab one that matches the provided objecttype URL if possible, now you can end up in a situation that you pick "the wrong" group.

Keep in mind this will especially happen on environments like DH where they have old exports that are being imported into an environment that has (an) existing group(s)